assets/pages/digital-subscription-checkout/digitalSubscriptionCheckout.jsx

```diff
@@ -18,12 +19,12 @@ import SubscriptionFaq from 'components/subscriptionFaq/subscriptionFaq';
 
 import { initReducer } from './digitalSubscriptionCheckoutReducer';
 import CheckoutStage from './components/checkoutStage';
-
 import './digitalSubscriptionCheckout.scss';
 
 // ----- Internationalisation ----- //
 
-const countryGroupId: CountryGroupId = detect();
+const countryGroupId: CountryGroupId = detectCountryGroup();
```

Comment:
I don't really want to call these functions here. I'd prefer to use the common state in digitalSubscriptionCheckoutReducer, but I am struggling to work out how to wire this up...